### PR TITLE
Add post with migration from Gatsby to Astro

### PR DESCRIPTION
**Pull request by helmerdavila**
#### Description (required)

- Add another guide about migrating from Gatsby to Astro
- Translating a single sentence

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

**Comment by liruifengv** [comment]
Thanks for your first contribution @helmerdavila 

Our best practice is to update the English page and other language pages separately. Can you update only the English page and create a new pr for `es`?

**Comment by helmerdavila** [comment]
> Thanks for your first contribution @helmerdavila
> 
> Our best practice is to update the English page and other language pages separately. Can you update only the English page and create a new pr for `es`?

Sure. I can get back the sentence that I translated.

But what about my spanish link to the blog post? Should I add it in a next PR as you said? Because the link points to a specific spanish version

File: src/content/docs/es/guides/migrate-to-astro/from-gatsby.mdx [hunk 1]
```diff
@@ -573,6 +573,7 @@ Este diseño de página muestra un encabezado cuando se visita la página de ini
 
 - Artículo de blog: [Otra migración: de Gatsby a Astro](https://logarithmicspirals.com/blog/migrating-from-gatsby-to-astro).
 
+
```
Comment (liruifengv) [inline src/content/docs/es/guides/migrate-to-astro/from-gatsby.mdx:576]:
```suggestion
```

**Review by liruifengv — APPROVED** [review]
Now LGTM! Thanks @helmerdavila Welcome to Astro team docs.

**Review by sarah11918 — APPROVED** [review]
This is great! Happy to have another resource here. Thanks for contributing, and welcome to Team Docs!  🥳

**Comment by sarah11918** [comment]
!coauthor

**Pull request merged**